content/ai/concepts/neural-networks/terms/recurrent-neural-networks/recurrent-neural-networks.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'Recurrent Neural Networks'
+Description: 'Recurrent Neural Network is a type of neural network distinguished by storing and re-using the output from previous steps as an additional input in the current step'
+Subjects:
+  - 'Machine Learning'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Neural Networks'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/machine-learning-engineer'
+---
+
+## What Is A Recurrent Neural Network (RNN)?
```

Comment:
```suggestion
```
Remove this header. In docs we don't use headers above the first paragraph. The paragraph comes right after the title.